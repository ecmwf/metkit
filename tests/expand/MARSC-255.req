retrieve,
    date=20250416,
    area=75/60/10/-20,
    grid=2.5/2.5,
    levtype=ml,
    levelist=50,
    param=248,
    interpolation=bilinear,
    target=data.reference 