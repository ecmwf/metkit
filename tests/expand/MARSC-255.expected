RETRIEVE,
    CLASS      = OD,
    TYPE       = AN,
    STREAM     = OPER,
    EXPVER     = 0001,
    REPRES     = SH,
    LEVTYPE    = ML,
    LEVELIST   = 50,
    PARAM      = 248,
    DATE       = 20250416,
    TIME       = 1200,
    STEP       = 0,
    DOMAIN     = G,
    TARGET     = data.reference,
    RESOL      = AUTO,
    INTERPOLATION = BILINEAR,
    AREA       = 75/60/10/-20,
    GRID       = 2.5/2.5 